pasqal_cloud/batch.py

```diff
@@ -51,30 +54,45 @@ class Batch:
     user_id: int
     priority: int
     status: str
-    webhook: str
+    webhook: Optional[str]
     _client: Client
     sequence_builder: str
-    start_datetime: Optional[str] = None
-    end_datetime: Optional[str] = None
+    start_datetime: Optional[str]
+    end_datetime: Optional[str]
     device_status: Optional[str] = None
```

Comment:
Nit: If you're removing all these `= None`, let's remove that one, too :)

Reply:
I only have one doubt here
`Optional[str]` is often expected to be handled automagically by inserting `None` if a value wasn't supplied. This is not how Python behaves though.  You'll still have to supply a value :P

Personally I've always preferred having these as `sart_datetime: str = ""`, since empty string in python evaluates as `False`

Reply:
sounds fine to me to leave it has you have then